preferencesfx/src/main/java/com/dlsc/preferencesfx/view/PreferencesFxDialog.java

```diff
@@ -119,6 +119,7 @@ private void setupDebugHistoryTable() {
   private void setupButtons() {
     LOGGER.trace("Setting Buttons up");
     final Button closeBtn = (Button) lookupButton(closeWindowBtnType);
+    closeBtn.setText("Save & Close");
```

Comment:
This is something we discussed a lot... We chose not to use "Save" here, since we thought it would imply that the changed preferences would only be applied when pressing this button. This is not the case, since by default changes in settings result in instant persistence in the application, while you do have a good point here, since they are truly "saved" to the preferences only upon pressing this button. I think @dlemmermann will probably have the last word on this.
If we decide to change the naming, this implementation is a bit limiting, since by default JavaFX ButtonTypes are i18n. Using this implementation means the button would show "Save & Close" in every language, so we would need to provide a possibility to i18n it as well - which is why I would suggest to stick to ButtonTypes for simplicity.

Reply:
Simply use a default close button (`ButtonType.CLOSE`), because that's the only thing it does.